paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/FlinkConnectorOptions.java

```diff
@@ -80,7 +80,16 @@ public class FlinkConnectorOptions {
                     .withDescription(
                             "Define a custom parallelism for the scan source. "
                                     + "By default, if this option is not defined, the planner will derive the parallelism "
-                                    + "for each statement individually by also considering the global configuration.");
+                                    + "for each statement individually by also considering the global configuration. "
+                                    + "If user enable the scan.infer-parallelism, the planner will derive the parallelism by infer-parallelism.");
```

Comment:
```suggestion
                                    + "If user enable the "+ INFER_SCAN_PARALLELISM.key() +", the planner will derive the parallelism by infer-parallelism.");
```

Reply:
the `INFER_SCAN_PARALLELISM` is defined after `SCAN_PARALLELISM`,so we can not add the `INFER_SCAN_PARALLELISM`  to the description of `SCAN_PARALLELISM`.